commands/utils/packagehandlers/gopackagehandler.go

```diff
@@ -9,7 +9,9 @@ type GoPackageHandler struct {
 	GenericPackageHandler
 }
 
-func (golang *GoPackageHandler) UpdateImpactedPackage(impactedPackage string, fixVersionInfo *utils.FixVersionInfo, extraArgs ...string) error {
+func (golang *GoPackageHandler) UpdateImpactedPackage(impactedPackage string, fixVersionInfo *utils.FixVersionInfo, extraArgs ...string) (shouldFix bool, err error) {
 	impactedPackage = strings.Trim(impactedPackage, "v")
+	// In Golang, we can address every package as direct because of the way that 'go get' works
```

Comment:
Here we can talk about the implementation @eyalbe4 

i think it's okay to address in Go to every package as direct when we are fixing a package.

Changing the DirectDependecy attribute in FixVersionInfo is not good because it's package handler related and not generic enough like isDirect or not.